src/eos/types/wire.cpp

```diff
@@ -242,7 +242,7 @@ wire& wire::replace(const wire& substr, const wire& newsubstr)
 
 }
 
-wire wire::masking(wire mask, int unmaskedLeft, int unmaskedRight) const
+wire wire::masking(wire mask, size_t unmaskedLeft, size_t unmaskedRight) const
```

Comment:
Here is a possible problem with condition **(remains > 0)**, because if it is always true for size_t.
So, the comparison should be changed onto **if (unmaskedPart < m_container.length())**